config.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"os"
+
+	"github.com/iver-wharf/wharf-core/pkg/config"
+	"github.com/iver-wharf/wharf-core/pkg/env"
+)
+
+// Config holds all configurable settings for wharf-provider-azuredevops.
+//
+// The config is read in the following order:
+//
+// 1. File: /etc/iver-wharf/wharf-provider-azuredevops/config.yml
+//
+// 2. File: ./wharf-provider-azuredevops-config.yml
+//
+// 3. File from environment variable: WHARF_CONFIG
+//
+// 4. Environment variables, prefixed with WHARF_
+//
+// Each inner struct is represented as a deeper field in the different
+// configurations. For YAML they represent deeper nested maps. For environment
+// variables they are joined together by underscores.
+//
+// All environment variables must be uppercased, while YAML files are
+// case-insensitive. Keeping camelCasing in YAML config files is recommended
+// for consistency.
+type Config struct {
+	API  WharfAPIConfig
+	HTTP HTTPConfig
+	CA   CertConfig
+}
+
+// WharfAPIConfig holds settings for the connection to the Wharf API.
+type WharfAPIConfig struct {
+	// URL is the base URL targetted towards the Wharf API. In a standard
+	// installation of Wharf, this would include the trailing "/api" in the URL
+	// path.
+	//
+	// Added in v1.3.0.
+	URL string
+}
+
+// HTTPConfig holds settings for the HTTP server.
+type HTTPConfig struct {
+	CORS CORSConfig
+
+	// BindAddress is the IP-address and port, separated by a colon, to bind
+	// the HTTP server to. An IP-address of 0.0.0.0 will bind to all
+	// IP-addresses.
+	//
+	// For backward compatibility, that may be removed in the next major release
+	// (v3.0.0), the environment variable BIND_ADDRESS, which was added in
+	// v2.0.0, will also set this value.
+	//
+	// Added in v1.3.0
+	BindAddress string
+}
+
+// CORSConfig holds settings for the HTTP server's CORS settings.
+type CORSConfig struct {
+	// AllowAllOrigins enables all
```

Comment:
It seems like part of the comment has gotten chopped here.

Reply:
Applying same suggestion as from https://github.com/iver-wharf/wharf-provider-gitlab/pull/23#pullrequestreview-736934503

```suggestion
	// AllowAllOrigins enables CORS and allows all hostnames and URLs in the
	// HTTP request origins when set to true. Practically speaking, this
	// results in the HTTP header "Access-Control-Allow-Origin" set to "*".
```